op-test

```diff
@@ -96,7 +96,7 @@ from testcases import OpalErrorLog
 from testcases import LightPathDiagnostics
 from testcases import DPO
 from testcases import EPOW
-from testcases import OpTestKernel
+from testcases import PowerNVDump
```

Comment:
You have to fix "class CrashSuite" as well.

Additionally you have to add FADUMP tests to crash_suite(0
  ... If fadump supported test fadump tests.. else skip them
  ... if kdump supported test kdump tests.. else skip them

Finally  better s/PowerNVDump.py/OpTestDump.py/ ?

-Vasant

Reply:
it was part of old file which was moved to PowerNVDump.. sure i can add fadump to that..

Reply:
I have added FAdump tests to crash_suite() and taken care of skipping tests when respective options are not enabled.